src/main/java/Duke.java

```diff
@@ -1,10 +1,20 @@
+import joi.Joi;
+import joi.utils.InvalidCommandException;
+import joi.utils.InvalidInputException;
+
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws IOException {
+        Joi joi = new Joi();
+        try {
+            joi.run();
+        }
+        catch (InvalidInputException e) {
+            System.err.println("OOPS!!! I'm sorry, but I don't know what " + e.getWrongInput() + " means :-(");
+        }
+        catch (InvalidCommandException e) {
+            System.err.println("OOPS!!! " + e.getWrongInput());
+        }
     }
 }
```

Comment:
I'm a bit confused why do we have both Duke.java and Joe.java. Maybe Duke can be merged with Jio?